test/vc-examples.spec.js

```diff
@@ -3,12 +3,17 @@
  */
 import chai from 'chai';
 import {credentialsDir} from '../lib/index.js';
+import fs from 'node:fs';
 
 const should = chai.should();
 
 describe('credentialsDir', () => {
   it('should exist path to credentials directory.', async () => {
     should.exist(credentialsDir);
     credentialsDir.should.be.a('string');
+
+    const exists = fs.existsSync(credentialsDir);
+
+    exists.should.equal(true);
```

Comment:
Could make this a one-liner:
```suggestion
    fs.existsSync(credentialsDir).should.be.true();
```

Reply:
Updated [6f3a88e](https://github.com/credential-handler/vc-examples/pull/8/commits/6f3a88e20b55fa8f668753fa24b9a006f68ea4f2)